### PR TITLE
[C-1874] Improve lineup loading animations, add fast-image

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

- Improves lineup loading animations to mimic mobile-web
- Improves image loading performance using react-fast-image

### Dragons

Should be rebased with @sliptype changes

### CODE REVIEW

File: packages/mobile/package.json [hunk 1]
```diff
@@ -112,6 +112,7 @@
     "react-native-draggable-flatlist": "3.0.7",
     "react-native-dynamic": "1.0.0",
     "react-native-fast-crypto": "2.2.0",
+    "react-native-fast-image": "8.6.3",
```
Comment (sliptype) [inline packages/mobile/package.json:115]: Let's keep an eye on our android crash logs when this goes out. I have a feeling we might see some issues

**Review by sliptype — APPROVED** [review]
Approving but let's wait to merge until after release is cut!

File: packages/mobile/package.json (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/package.json:115]: sounds like a plannn

**Review by raymondjacobson — APPROVED** [review]
Looking forward to having this in hand and seeing!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-1874-fast-image

**Pull request merged**